pkg/start/start.go

```diff
@@ -195,6 +192,15 @@ func (o *Options) run(ctx context.Context, controllerCtx *Context, lock *resourc
 	controllerCtx.OpenshiftConfigManagedInformerFactory.Start(informersDone)
 	controllerCtx.InformerFactory.Start(informersDone)
 
+	if !cache.WaitForCacheSync(informersDone, controllerCtx.CVO.GetClusterVersionCacheSynced()) {
```

Comment:
[`CVInformerFactory`][1] is an [`externalversions.SharedInformerFactory`][2].  Maybe we can use:

```go
if !controllerCtx.CVInformerFactory.WaitForCacheSync(informersDone) {
```

here, and not need the new `GetClusterVersionCacheSynced`?

[1]: https://github.com/openshift/cluster-version-operator/blob/b133d822ea2de7ccf7f9b6ee5530da703f3d0b2c/pkg/start/start.go#L414
[2]: https://pkg.go.dev/github.com/openshift/client-go/config/informers/externalversions#SharedInformerFactory

Reply:
That works